packages/core/src/components/Price/index.tsx

```diff
@@ -43,14 +43,20 @@ function Price({
   let currency = baseCurrency;
   let micros = baseMicros;
 
-  if (baseAmount === undefined || baseAmount === null) {
+  if (
+    baseAmount === undefined ||
+    baseAmount === null ||
+    (typeof baseAmount === 'string' && isNaN(Number(baseAmount)))
```

Comment:
`Number.isNaN`

Reply:
@milesj going to leave these as is, it's a little cleaner:

`Number.isNaN('[Hidden]')` -> `false`
`Number.isNaN(Number('[hidden]'))` -> `true`
`isNaN(Number('[Hidden]'))` -> `true`